Bottom: 5b26a3acf2bf52a41857706b01c7e8c49181358c
Top:    3066dabea7e009cb81cfcb1fd6865005ec35b09a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-24 16:30:47 -0500

Remove libicu52 from Dockerfile

Newest upstream versions of php:5.6-apache are based on debian stretch and 
use libicu57, which is already installed. Besides this, libicu-dev would 
already install libicuXX.


---
diff --git a/docker/docker-mediawiki/Dockerfile b/docker/docker-mediawiki/Dockerfile
index 6aaefd61..54421165 100644
--- a/docker/docker-mediawiki/Dockerfile
+++ b/docker/docker-mediawiki/Dockerfile
@@ -13,7 +13,6 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         git \
         imagemagick \
-        libicu52 \
         libicu-dev \
         libldap2-dev \
     && \
